examples/simple-connect/index.js

```diff
@@ -18,16 +14,11 @@ function App() {
   const activate = async connector => {
     setLastError('')
 
-    try {
-      await wallet.activate(connector)
-    } catch (err) {
-      if (err instanceof RejectedActivationError) {
-        setLastError('Connection error: the user rejected the activation')
-      }
-      setLastError(err.message)
-    }
+    await wallet.connect(connector)
   }
 
+  useEffect(() => setLastError(wallet.error?.name ?? ''), [wallet])
```

Comment:
Just curious, should we continue to reference `message` in the examples? it feels like it might be a more useful descriptor.

Reply:
Also wondering if we should keep one of the example with custom error messages, to demonstrate how to check for a specific error type using `typeof`?